pygmt/src/ternary.py

```diff
@@ -0,0 +1,59 @@
+"""
+ternary - Plot data on ternary diagrams.
+"""
+from pygmt.clib import Session
+from pygmt.helpers import build_arg_string, fmt_docstring, kwargs_to_strings, use_alias
+
+
+@fmt_docstring
+@use_alias(
+    B="frame",
+    C="cmap",
+    F="center",
+    G="fill",
+    JX="projection",
+    R="region",
+    S="style",
+    U="timestamp",
+    V="verbose",
+    W="pen",
+    X="xshift",
+    Y="yshift",
+    p="perspective",
+    t="transparency",
+)
+@kwargs_to_strings(R="sequence", T="sequence", c="sequence_comma", p="sequence")
+def ternary(self, table, **kwargs):
+    r"""
+    Plots a histogram, and can read data from a file or
+    list, array, or dataframe.
```

Comment:
Probably need to make this docstring more Pythonic, but here's a first shot copied from https://github.com/GenericMappingTools/gmt/blob/6.3.0/doc/rst/source/ternary.rst
```suggestion
    Reads (*a*,\ *b*,\ *c*\ [,\ *z*]) records from *table* and plots symbols at
    those locations on a ternary diagram. If a symbol is selected and no symbol
    size given, then we will interpret the fourth column of the input data as
    symbol size. Symbols whose *size* is <= 0 are skipped. If no symbols are
    specified then the symbol code (see **style** below) must be present as
    last column in the input.  If **style** is not specified then we instead
    plot lines or polygons.
```